layers/+lang/go/funcs.el

```diff
@@ -66,3 +67,4 @@
 (defun spacemacs/go-packages-gopkgs ()
   "Return a list of all Go packages, using `gopkgs'."
   (sort (process-lines "gopkgs") #'string<))
+
```

Comment:
Remove this. No blank lines at the end of the file.

Reply:
OK, done. sorry for that

Reply:
No problem at all :)